pounders/m/pounders.m

```diff
@@ -81,6 +81,11 @@
     Model.Par(5) = 0;     % [log] Flag to find affine points in forward order (0)
 end
 
+% Casting to integers here prevents rare issues with non-integer arguments for
+% reshape on some GH Action instances.  See Issue #237 for more details.
+n = int32(n);
+m = int32(m);
```

Comment:
Should the casts be moved higher.  For example, before `np_max` might be set using `n`?